src/views/WorkflowsTable.vue

```diff
@@ -173,7 +174,7 @@ export default {
   },
   methods: {
     viewWorkflow (workflow) {
-      this.$router.push({ path: `/workflows/${workflow.tokens.workflow}` })
+      this.$router.push({ path: `/workspace/${workflow.tokens.workflow}` })
```

Comment:
Is there a way to use `src/router/paths` directly, to avoid the possibility of breaking these links?

Reply:
No idea.